modules/transactionpool/accept.go

```diff
@@ -235,5 +235,10 @@ func (tp *TransactionPool) RelayTransaction(conn modules.PeerConn) error {
 	if err != nil {
 		return err
 	}
-	return tp.AcceptTransaction(t)
+	err = tp.AcceptTransaction(t)
+	// ErrTransactionPoolDuplicate is benign
+	if err == modules.ErrTransactionPoolDuplicate {
+		err = nil
```

Comment:
only in RelayTransaction. The main reason I took it out is because it was clogging gateway.log with "errors." Receiving a duplicate is expected behavior so I decided to not count it as an error. Ditto for `ErrKnownBlock` above.